assets/scss/style.scss

```diff
@@ -0,0 +1,835 @@
+        margin: 0;
+        color: #87879d;
+        font-size: 16px;
+        line-height: 26px;
+      }
+    }
+    .blockImage {
+      margin: 64px auto 0px auto;
+      display: grid;
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+      grid-template-rows: 1fr;
+      gap: 0px 11px;
+      grid-template-areas: "videocall videochatting meeting texting";
+      .videocall {
+        grid-area: videocall;
+      }
+      .videochatting {
+        grid-area: videochatting;
+      }
+      .meeting {
+        grid-area: meeting;
+      }
+      .texting {
+        grid-area: texting;
+      }
+      img {
+        width: 250px;
+        height: 242px;
+        border-radius: 8px;
+      }
+    }
+    .paragraph2 {
+      margin: 80px auto 0px auto;
+      text-align: center;
+
+      h2 {
+        font-size: 16px;
+        line-height: 26px;
+        font-weight: 900;
+        color: #4d96a9;
+        text-transform: uppercase;
+      }
+      h1 {
+        margin: 16px auto 32px auto;
+        overflow: hidden;
+        font-size: 40px;
+        line-height: 44px;
+        font-weight: 900;
+        width: 445px;
+      }
+      p {
+        font-size: 18px;
+        line-height: 26px;
+        font-weight: 500;
+        color: #87879d;
+        margin: 0px auto 0px auto;
+        width: 540px;
+      }
+    }
+    .wrapper {
+      .verticalLine {
+        height: 84px;
+      }
+    }
+    .footer {
+      text-align: center;
+      width: 100%;
+      background: linear-gradient(
+        rgba(77, 150, 169, 0.9),
+        rgba(77, 150, 169, 0.9)
+        ),
+        url(/assets/mobile/image-footer.jpg);
+        background-size: cover;
+        .paragraph3 {
+        width: 1110px;
+        height: 308px;
+        margin: auto;
+        display: flex;
+        align-items: center;
+        h1 {
+          width: 350px;
+          text-align: left;
+          overflow: hidden;
+          color: white;
+          font-size: 40px;
+          line-height: 44px;
+          font-weight: 900;
+        }
+        p {
+          width: 350px;
+          margin: 0px 125px 0px 92px;
+          text-align: left;
+          color: #fafafa;
+          font-size: 18px;
+          line-height: 26px;
+          font-weight: 500;
+        }
+        button {
+          width: 193px;
+          height: 58px;
+          border-radius: 29px;
+          font-size: 16px;
+          line-height: 26px;
+          background-color: #855fb1;
+          border: none;
+          font-weight: 900;
+          color: white;
+          &:hover {
+            cursor: pointer;
+            background-color: #B18BDD;
+          }
+          span {
+            margin-left: 4px;
+            color: #D9B8FF;
+          }
+        }
+      }
+    }
+  }
+}
+
+// // Tablet Portrait & Landscape
+
+@media screen and (min-width: 480px) and (max-width: 1023px) {
+  .container {
+    .logo {
+      margin-top: 48px;
+    }
+    .imageSlider {
```

Comment:
Stilove za `.imageSlider` imas u vise fajlova. To nije dobro jer se takav kod tesko odrzava...
Znaci tamo gde zelis da ga definises tamo odmah dodajes i responsive definicije